docs/CONFIG.md

```diff
@@ -38,30 +38,23 @@ Configuration lives in [config.yml](../config.yml).
-- `connector_id`: The ID of the custom connector.
-- `service_type` The service type of the custom connector.
+- `connector_id`: The ID of the connector.
+- `service_type` The service type of the connector.
 - `sources`: A mapping/dictionary between service type and [Fully Qualified Name
 (FQN)](https://en.wikipedia.org/wiki/Fully_qualified_name). E.g. `mongodb: connectors.sources.mongo:MongoDataSource`.
 
 ## Run the connector service on Elastic Cloud
 
-When you have an Enterprise Search deployment on Elastic Cloud post 8.5.0, the connector service is automatically deployed. The connector service can only run in native mode on Elastic Cloud (i.e. it will only sync data for connectors with service type listed in `native_service_types`), and the Elasticsearch connection configurations (i.e. `host`, `username`, `password`) will be overridden, and a special Cloud user `cloud-internal-enterprise_search-server` will be used for Elasticsearch connection, which will have proper privilege on the connector index (`.elastic-connectors`), the connector job index (`.elastic-connectors-sync-jobs`) and the connector content indices (`search-*`).
+When you have an Enterprise Search deployment on Elastic Cloud post 8.5.0, the connector service is automatically deployed.
+The connector service runs in native mode on Elastic Cloud, and the Elasticsearch connection configurations (i.e. `host`, `username`, `password`) are automatically configured.
+A special Cloud user (`cloud-internal-enterprise_search-server`) is used for the Elasticsearch connection.
+This user has proper privileges on the connector index (`.elastic-connectors`), the connector job index (`.elastic-connectors-sync-jobs`) and the connector content indices (`search-*`).
 
-## Run the connector service on-prem
-
-### Run the connector service in native mode
+If you wish to connect more connector types than are natively available on Elastic Cloud, you will need to run the connector service on-prem, as described below.
 
-1. Make sure the service types of supported native connectors are configured in `native_service_types`.
-2. Configure the Elasticsearch connection, with basic auth (`username` and `password`), or with API key (generated via _Stack Management_ > _Security_ > _API keys_ > _Create API key_). Make sure the user or the API key has at least the privileges to `manage`, `read` and `write` the connector index (`.elastic-connectors`), the connector job index (`.elastic-connectors-sync-jobs`) and the connector content indices (`search-*`).
-3. Run the connector service with
-    ```shell
-    make run
-    ```
-
-### Run the connector service for a custom connector
+## Run the connector service on-prem
 
-1. Go to Kibana, _Enterprise Search_ > _Create an Elasticsearch index_. Use the `Build a connector` option for an ingestion method to create an index.
+1. Go to Kibana, _Enterprise Search_ > _Create an Elasticsearch index_. Select the type of connector you wish to use.
```

Comment:
The on-prem connector service will work for:
1. a connector client
2. a customized connector
3. a converted native connector

Probably we should make this step more general:

```suggestion
When you have a connector client, a customized connector, or a converted native connector, you have to run the connector service on-prem.

1. In Kibana, go to the `Configuration` tab of the connector.
```

Reply:
Driving by here, but I think a connector client is what we call a self-managed connector. So you use the on-prem connector service to run connector clients.

Whether these are customized connectors or converted native connectors is perhaps irrelevant here. 🤷

Reply:
I can work on the phrasing here to capture both points. That any "converted native connector," "net-new custom connector" or "customized/modified connector" must be run as a connector client, on-prem. WDYT?

Reply:
👍 

FYI I wonder if we could rename connector clients as self-managed connectors? It would be so much clearer. We've done the hard work of removing confusion around "native"!

Reply:
I would prefer that. But I think that @danajuratoni has been very intentional about our taxonomy thus-far, so I'm hesitant to advocate too strongly.

Reply:
Indeed, it's not easy to backtrack and rename stuff